Mu2eG4/inc/Mu2eG4Config.hh

```diff
@@ -118,7 +118,8 @@ namespace mu2e {
       fhicl::Atom<double> protonProductionCut {Name("protonProductionCut")};
 
       fhicl::Atom<std::string> physicsListName {Name("physicsListName")};
-      fhicl::Atom<bool> turnOffRadioactiveDecay {Name("turnOffRadioactiveDecay"), false};
+      fhicl::Atom<bool> turnOffRadioactiveDecay {Name("turnOffRadioactiveDecay"),
+          Comment("turnOn/Off radioactive decay. If both are false, we get the G4 default, which may be physics list dependent. If both are true, it is a configuration error."), false};
       fhicl::Atom<bool> turnOnRadioactiveDecay {Name("turnOnRadioactiveDecay"), false};
```

Comment:
I suggest adding a comment here too.  Maybe the same comment?  Or just refer to the above comment?

Reply:
The --print-description output looks like:

             ## turnOn/Off radioactive decay. If both are false, we get the G4 default, which may be physics list dependent. If both are true, it is a configuration error.

              turnOffRadioactiveDecay: false  # default

              turnOnRadioactiveDecay: false  # default

Do you know how to add a properly indented line break?

Reply:
I guessed the line break correctly.  More in a minute.

Reply:
Done and pushed.